addons/web/static/src/views/kanban/kanban_model.js

```diff
@@ -270,7 +270,16 @@ class KanbanGroup extends Group {
     updateAggregates(groupData) {
         const fname = this.groupByField.name;
         const { sumField } = this.model.progressAttributes;
-        const group = groupData.find((g) => this.valueEquals(g[fname]));
+        const group = groupData.find((g) => {
```

Comment:
It feels as if these changes are related to both the MRR feature and the forecast feature.

Maybe we should split into different commits to explain a bit more the changes introduced?

Reply:
This is actually a bugfix, it should be in a separate commit as well.